utils/docker/Dockerfile.centos.7

```diff
@@ -47,45 +47,41 @@ LABEL maintainer="daos@daos.groups.io"
 ARG UID=1000
 
-# for good measure, clean the metadata
-RUN yum clean metadata
-
-# Install basic tools
-RUN yum -y install epel-release; \
-    yum -y install \
-        boost-devel clang-analyzer cmake CUnit-devel doxygen file flex \
-        gcc gcc-c++ git golang graphviz lcov                           \
-        libaio-devel libcmocka-devel libevent-devel libiscsi-devel     \
-        libtool libtool-ltdl-devel libuuid-devel libyaml-devel         \
-        make meson nasm ninja-build numactl-devel openssl-devel        \
-        pandoc patch python python-devel python36-devel python-magic   \
-        python-pep8 python-pygit2 python2-pygithub python-requests     \
-        readline-devel scons sg3_utils ShellCheck yasm pciutils        \
-        valgrind-devel python36-pylint man java-1.8.0-openjdk maven    \
-        json-c-devel python36-requests lz4-devel
-
-# DAOS specific
-RUN yum -y install \
-           python2-avocado python2-avocado-plugins-output-html \
-           python2-avocado-plugins-varianter-yaml-to-mux       \
-           python-pathlib                                      \
-           ndctl ipmctl e2fsprogs                              \
-           python2-clustershell python2-Cython                 \
-           python36-clustershell python36-paramiko             \
-           python36-numpy python36-jira python3-pip Lmod       \
-           fuse3-devel libipmctl-devel python36-pyxattr        \
-           hwloc-devel patchelf python36-tabulate              \
-           java-1.8.0-openjdk python-distro python36-distro
-RUN yum -y install openmpi3-devel
+# Install OS updates and package.  Include basic tools and daos dependencies
+# that come from the core repo.
+# Clean up any repos afterwards to save space.
+RUN yum clean all && yum -y install epel-release &&			\
+    yum -y upgrade &&							\
+    yum -y install							\
+	boost-devel clang-analyzer cmake CUnit-devel doxygen file flex	\
+	gcc gcc-c++ git golang graphviz lcov				\
+	libaio-devel libcmocka-devel libevent-devel libiscsi-devel	\
+	libtool libtool-ltdl-devel libuuid-devel libyaml-devel		\
+	make meson nasm ninja-build numactl-devel openssl-devel		\
+	pandoc patch python python-devel python36-devel python-magic	\
+	python-pep8 python-pygit2 python2-pygithub python-requests	\
+	readline-devel scons sg3_utils ShellCheck yasm pciutils		\
+	valgrind-devel python36-pylint man java-1.8.0-openjdk maven	\
+	json-c-devel python36-requests lz4-devel			\
+	python2-avocado python2-avocado-plugins-output-html		\
+	python2-avocado-plugins-varianter-yaml-to-mux			\
+	python-pathlib							\
+	ndctl ipmctl e2fsprogs						\
+	python2-clustershell python2-Cython				\
+	python36-clustershell python36-paramiko				\
+	python36-numpy python36-jira python3-pip Lmod			\
+	fuse3-devel libipmctl-devel python36-pyxattr			\
+	hwloc-devel patchelf python36-tabulate				\
+	java-1.8.0-openjdk python-distro python36-distro install	\
```

Comment:
install?

Reply:
good spot, I'll back this out and check the output for other possible packages it doesn't recognise.